### ISSUE TITLE
Wrong folder deletion

### ISSUE THREAD

**Issue by selsie** [Synthesis-Collective/SynthEBD#126]
I'm deleting one of the installed texturepacks, and the standalone patcher offers to delete it's folder. But it is the folder of another mod!

![grafik](https://github.com/Synthesis-Collective/SynthEBD/assets/15048308/fe71cca2-7aef-476d-bc30-36f6c610c6f6)

**Comment by Piranha91** [comment]
Yeah that’s been on my backburner for a while. I was originally thinking of having SynthEBD just keep a database of which config’s textures were installed to which folder, but that would break if the user installs and then renames the mod folder. So instead it looks through all mod folders and tries to see if any of them have any of the files referenced by the config. In this case the config must point to a texture that present in Hall of Forgotten. If you don’t mind, could you upload that config file so I can see which texture might have confused the matching function?

**Comment by selsie** [comment]
What I was doing at the time:
I was installing the Babygirl HD Skin, and got an error that basically all texture files are missing (a long list), AND that there is no folder specified. I thought it was an error on my side, and deleted the textures set, when this popped up. I restarted SynthEBD (standalone) and tried the installation again, when I got the same delete message (and the same missing textures) again. 

It's the Baby Girl HD Fair Skin - UNP - SynthEBD Config from https://www.nexusmods.com/skyrimspecialedition/mods/98534

The only textures in that folder are facegen textures:
![grafik](https://github.com/Synthesis-Collective/SynthEBD/assets/15048308/7ab1383c-5d19-4e5b-b025-4a6d73e7ef37)

To "tag" the config texture folders, why not add a file with the name of the config as filename / directory name, so even if the folder is renamed, the folder will be found.

**Comment by selsie** [comment]
The manifest.json file is lacking some information, the Babygirl HD Skin.json in that archive also looks weird. I think the deletion of the wrong folder may also be caused by that.

BTW: 
![grafik](https://github.com/Synthesis-Collective/SynthEBD/assets/15048308/73e958c9-f7a6-4efe-a357-1cf7a7bfa293)
I got this message, and no folder was created at all.

**Comment by selsie** [comment]
All except the esp.

![grafik](https://github.com/Synthesis-Collective/SynthEBD/assets/15048308/9cd5b221-226c-43d1-bd1b-8ed11f741290)

**Comment by Piranha91** [comment]
Yeah I was thinking about it after you opened the issue - some meta file would be ideal. I’ll implement that in the next version. Sorry about your merge.

If you check the comments on the config file’s page, I’ve been trying to explain that almost all of them are improperly made. I never considered this particular consequence of the mistakes, but most of them will not work correctly. You’ll honestly have better luck just using the Config Drafter to make configs for those texture mods - it’s what I’ve been doing for my recent configs. For simple configs without a lot of texture variations it usually “just works”, and for more complex texture mods with a lot of variants it still takes care of most of the work and just requires a bit of fine tuning afterward.

**Comment by selsie** [comment]
I guess you are familiar with zMerge. I think it's a good idea to put the relevant configuration files right in the folder with the textures, so in future some other tool could also use the information and do something with it (thinking of the MO2 Plugin Merge Plugins Hide)

I'll have a look at the config drafter, but I am not really sure I know what to do. Skyrim Modding requires a PHD in several different disciplines right now :)

**Comment by Piranha91** [comment]
Yup you’re right - there’s a lot of things that SynthEBD has to do and only one of me, so some peripheral functions (like deleting assets) aren’t as well thought out or implemented. Happy to take suggestions like yours when issues come up.

I have a pretty detailed guide for the config drafter if you need it. It’s pretty straightforward, only takes a minute once you know what you’re doing. https://www.nexusmods.com/skyrimspecialedition/articles/5747

**Comment by selsie** [comment]
The detailed guide is just what the doctor ordered :)
Thanks for your help and dedication!

**Comment by selsie** [comment]
The detailed guide is just what the doctor ordered :)
Thanks for your help and dedication!

Fiddling around with the settings...
![grafik](https://github.com/Synthesis-Collective/SynthEBD/assets/15048308/3bf8aa16-98ef-4aad-bfb6-b49c994642b3)
These are actually toggles. Is it possible to disable a checkbox in the other set when checked? I can't allow and disallow the same thing, so they are mutually exclusive.

**Comment by Piranha91** [comment]
Addressed both complaints in the latest Synthesis version of the patcher (which gets constant updates as I make them). I'll leave this open for now until I release the standalone 1.0.1.9 version.

**Comment by selsie** [comment]
![grafik](https://github.com/Synthesis-Collective/SynthEBD/assets/15048308/7cb5a510-2705-445b-b2a7-0ca5b95ed7a4)
Where to set these values?

**Comment by Piranha91** [comment]
This will be in SynthEBD's O/AutoBody Integration Menu -> Body Descriptors -> (Age:Mature or Age: Younger) -> Distribution Rules. Those specific ones come from pamonha so I don't have them in my setup, but should look like this:

![image](https://github.com/Synthesis-Collective/SynthEBD/assets/63175798/7b066d62-f98d-4266-b6e1-d2326c1dad56)

I don't have any rules set up, but in his config file he must have either Allowed or Disallowed attributes set up. 

Someone has reported a similar issue a few months ago and I thought I fully addressed it, but I guess there's still a flaw. Can you walk me through what you did so I can try to replicate and fix? What do you mean by "I reimported all my files"? When you imported the "exported Body Slides.json", did you keep all of the default settings or did you uncheck any of the options?

**Comment by Piranha91** [comment]
Oh, he's saying you need to import bodyslides before installing any of the asset Config Files from his page. But that requirement is actually outdated as of a couple months ago - I'll leave a comment on his Nexus page.

I just tried importing his file, and there is a blank attribute that you'll need to delete:
![image](https://github.com/Synthesis-Collective/SynthEBD/assets/63175798/4d1fa8a1-f66d-4d6d-95c2-417a6c2d5142)

The "Mature" one looks fine though so something happened to it during installation that I/we will need to figure out.

When you say you re-imported the presets, do you mean that you first exported your existing presets, then imported Pamonha's configuration file, then imported your original presets from the file that you had first exported?
